ways_of_working/processes/version_control.md

```diff
@@ -0,0 +1,95 @@
+    - `chore` for branches handling technical maintenance such as editing documentation, adding tests, cleaning up code, changing version numbers, updating dependencies, etc.
+- `issue_number` should be the number identifying the GitHub issue this PR is addressing.
+- `short_description` should be a few words, separated by `-`, that quickly explain the topic of the branch.
+
+Here are a few examples:
+- `feature/3700-remove-unused-css`
+- `enhancement/2773-no-beta`
+- `fix/3437-picture-lazyload-exclusion`
+
+#### Branch protections
+
+Branch protections are rules that can prevent some events on specific branches. They can be configured for each repository on GitHub in the Settings of the repository.
+
+Our standard configuration is to protect both the `trunk` and the `develop` branches with the following rules:
+- Require a pull request before merging: all commits must be made to a non-protected branch and submitted via a pull request before they can be merged into a branch that matches this rule.
+    - Require 1 approval: pull requests targeting a matching branch require a number of approvals and no changes requested before they can be merged.
+    - Dismiss stale pull request approvals when new commits are pushed: New reviewable commits pushed to a matching branch will dismiss pull request review approvals.
+
+Those rules enforce the [peer-review process](reviews.md), ensuring the code we produce and released is at least validated by two developers.
+
+#### Closing branches
+
+Branch owners are responsible for ensuring their branches don't end up stale. If they are relevant, they should be merged and deleted, otherwise they can be closed and deleted. To merge a branch, our [review process](reviews.md) must be followed.
+
+Once a branch has been merged (or closed), it should be deleted. This ensures that we keep only branches for ongoing work, easing the identification of in progress development on a repository.
+
+### Commits
+
+Usually, a single developer is working on a branch. Therefore, it could seem sufficient to commit all changes once they are ready and working. However, this can cause troubles if one needs to circle back to the implementation to investigate a bug or understand what has been implemented. More numerous, small and well-scoped commits can help to navigate the changes, test intermediate versions for a bug, and eventually precisely rollback changes. Therefore, commits should rather be done frequently, and only contain a small, well-defined scope. 
+
+Commit messages are critical for teammates to navigate the history and quickly understand the reason of a change. The first line of a commit message is a brief summary of the commit, describing the expected result of the change or what is modified in the code. Then, a longer message can be added to provide more details if needed. This appended message must be separated from the first line by a blank line, to ensure the first line remains the highly visible description.
```

Comment:
I like to use the formula I've seen elsewhere that says the commit message should be the completion to the sentence "This commit will..."

It helps to have a standard way of writing the first line and it makes it easy to understand what the commit is about

Reply:
Ah, that's a nice way to put it, let me add it